### PR TITLE
Dev branch8

### LINKED ISSUE #6: Create results screen
A screen with the score.

We could use the game screen and either :-
1, On a win = swap falling player sprite with one with an open parachute with "Winner" text
2, On a loss = turn physics back on the player character and let it fall through the bottom of the screen, with "Failed" text

### PR DESCRIPTION
**Pull request by Shabucky1812**
Added simple end screens for winning and losing. More features can be added, this is just a good foundation.

### CODE REVIEW

**Comment by KelvinHere** [comment]
That's perfect

**Comment by KelvinHere** [comment]
Great stuff

**Review by KelvinHere — APPROVED** [review]
